docs/man_pages/project/testing/livesync-ios.md

```diff
@@ -1,20 +1,38 @@
 livesync ios
 ==========
 
-Usage | Syntax
+Usage | Synopsis
 ------|-------
 General | `$ appbuilder livesync ios [--device <Device ID>] [--companion] [--watch]`
 
 Synchronizes the latest changes in your project to connected iOS devices.
 
+<% if(isConsole) { %>
+<% if(isMobileWebsite) { %>
+WARNING: This command is not applicable to mobile website projects. To view the complete help for this command, run `$ appbuilder help livesync ios`
+<% } %>
+<% if(isNativeScript) { %>
+WARNING: This command is not applicable to NativeScript projects. To view the complete help for this command, run `$ appbuilder help livesync ios`
```

Comment:
Waiting for further confirmation on this one as the document here shows it as not working. https://docs.google.com/spreadsheets/d/1wKhY9QYNI0E2H1pp5-L7y48gzyLL8GXAmduoGVkMDbw/edit#gid=692116110